DUI3-DX/Connectors/ArcGIS/Speckle.Connectors.ArcGIS3/Bindings/ArcGISReceiveBinding.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Speckle.Connectors.ArcGIS.Utils;
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Connectors.DUI.Bridge;
+using Speckle.Connectors.DUI.Models.Card;
+using Speckle.Connectors.Utils.Cancellation;
+using Speckle.Connectors.Utils.Operations;
+using Speckle.Core.Logging;
+using ICancelable = System.Reactive.Disposables.ICancelable;
+
+namespace Speckle.Connectors.ArcGIS.Bindings;
+
+public sealed class ArcGISReceiveBinding : IReceiveBinding, ICancelable
+{
+  public string Name { get; } = "receiveBinding";
+  private readonly CancellationManager _cancellationManager;
+  private readonly ArcGISDocumentStore _store;
+  private readonly ReceiveOperation _receiveOperation;
```

Comment:
as you do your unit of work subtask, you'll need to inject the factory for that here and resolve the ReceiveOperation in the Receive method.

Reply:
TBD in UOW ticket